eth2/utils/int_to_bytes/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copied from sigp/lighthouse with some modifications and deletions
```

Comment:
sigp/lighthouse is licensed under Apache-2.0, which means you need to include the license document, at the very least. More information: https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)

Reply:
Awesome, thanks, I'll update this.